algosdk/transaction.py

```diff
@@ -520,6 +520,18 @@ def __eq__(self, other):
             and self.sprfkey == other.sprfkey
         )
 
+    @staticmethod
+    def _fixed_bytes64(key, size):
+        if key is None:
+            return None
+        if isinstance(key, (bytes, bytearray)) and len(key) == size:
+            return base64.b64encode(key)
+        if len(base64.b64decode(key)) == size:
```

Comment:
maybe also add `isinstance(key, str)` otherwise `b64decode` will probably throw an exception (did not check tho)

Reply:
I suppose it will, but that's life in a dynamic language, I think.  Before you could have stuffed something crazy in that field, and you'd get an exception somewhere when we tried to serialize.  Now it happens earlier, which I think is better.

But I don't want to try to turn Python into Go and insert type checks all over the place.